codespell_lib/data/dictionary.txt

```diff
@@ -27729,14 +27869,26 @@ hopyfully->hopefully
 horicontal->horizontal
 horicontally->horizontally
 horinzontal->horizontal
+horison->horizon
```

Comment:
Perhaps `orison`, although rare?

Same for plural.

Reply:
I couldn't find this typo.